<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel</class>
 <widget class="QMainWindow" name="Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QPushButton{
min-width:160px;
min-height:36px;
font-family:&quot;微软雅黑&quot;;
font-size:14px;
background-color: rgb(255, 255, 255);
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:32px;
color: yellow;
}
QLCDNumber{
min-width:260px;
min-height:75px;
border:1px solid white;
color:white;
padding:1px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbl_gameLevel">
         <property name="text">
          <string>第一关</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_gameScore">
         <property name="text">
          <string>得分</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLCDNumber" name="lcd_gameScore"/>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_start">
         <property name="text">
          <string>开始游戏</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_pause">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_restart">
         <property name="text">
          <string>重新开始游戏</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_stop">
         <property name="text">
          <string>结束</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
